tracer/span.go

```diff
@@ -161,6 +161,11 @@ func (s *spanImpl) FinishWithOptions(opts opentracing.FinishOptions) {
 		}
 	}
 
+	if !s.raw.Context.IsSampled() || !*s.raw.Context.Sampled {
+		debugSpan := s.raw.Tags["debug"] == true
```

Comment:
Should we compare against both `true` and `"true"`?

I believe the opentracing tags support any value `map[string]interface{}`. So users may inadvertenty set this to a string. We should probably do the same for the error tag below as well while we are it.

Reply:
modified, and allowed case insensitive string comparison for true